src/test/testarb_2.arb

```diff
@@ -0,0 +1,11 @@
+{
+    "@@locale": "en",
+    "@@x-reference": false,
```

Comment:
Why is this false? Is it because the test overrides it (by simulating it being set in the yaml file)? If so, is this key necessary?

Should there be a test with is set to `true`?